Packs/Cymulate/Integrations/Cymulate_v2/Cymulate_v2.py

```diff
@@ -0,0 +1,1791 @@
+    if attack_payload and attack_payload.startswith('http'):
+        incident_data['url'] = attack_payload
+
+    elif attack_payload:
+        incident_data['attackType'] = attack_payload
+
+    incident = {
+        'name': extract_event_name(event, module_name),
+        'occurred': event_timestamp,
+        'severity': convert_to_xsoar_severity(event.get('Risk')),
+        'rawJSON': json.dumps(remove_empty_elements(incident_data)),
+    }
+
+    if not event_timestamp:
+        event_timestamp = event.get('Timestamp') if event.get('Timestamp') \
+            else event.get('Attack_Timestamp')
+    if validate_timestamp(event_timestamp):
+        occurred = datetime.strptime(event_timestamp, CY_GENERAL_DATE_FORMAT).strftime(DATE_FORMAT)
+        incident['occurred'] = occurred
+
+    return incident
+
+
+def convert_to_xsoar_severity(severity: Optional[Any]) -> int:
+    """Maps Cymulate severity to Cortex XSOAR severity.
+
+    Args:
+        severity(str): severity as returned from Cymulate API.
+
+    Returns:
+        int: Cortex XSOAR Severity (1 to 4)
+    """
+    if severity:
+        return {
+            'low': IncidentSeverity.LOW,
+            'medium': IncidentSeverity.MEDIUM,
+            'high': IncidentSeverity.HIGH,
+            'critical': IncidentSeverity.CRITICAL
+        }.get(severity.lower(), 0)
+    return 0
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication. Returning 'ok' indicates that the integration
+    works like it is supposed to. Connection to the service is successful.
+
+    Args:
+        client (Client): Cymulate client.
+    """
+    test_message = 'ok'
+    try:
+        response = client.validate()
+        if not response.ok:
+            raise DemistoException('Authorization Error: make sure API Key is correctly set.')
+
+    except DemistoException as err:
+        if 'Unauthorized' in str(err):
+            test_message = f'Authorization Error: make sure API Key is correctly set.\n\n{err}'
+        else:
+            raise err
+
+    return test_message
+
+
+def list_exfiltration_template_command(client: Client) -> CommandResults:
+    """Retrieve a list of all exfiltration templates.
+
+    Args:
+        client (Client): Cymulate client.
+
+    Returns:
+        CommandResults: A CommandResults object that is then passed to 'return_results',
+                        containing the list of all exfiltration templates.
+
+    """
+    raw_response = client.list_templates(ENDPOINT_DICT.get('exfiltration'))
+    outputs = raw_response.get('data', None)
+    readable_output = tableToMarkdown('Exfiltration templates list:', outputs, removeNull=True)
+
+    command_results = CommandResults(
+        outputs_prefix='Cymulate.Exfiltration.Templates',
+        outputs_key_field='_id',
+        readable_output=readable_output,
+        outputs=outputs,
+        raw_response=raw_response
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)
```suggestion
        raw_response=raw_response,
```